### PR TITLE
Added Python Solution for Family Tree (Bronze)

### PR DESCRIPTION
**Pull request by ryanchou-dev**
all test cases pass for this solution :)

### CODE REVIEW

File: solutions/usaco-833.mdx [hunk 1]
```diff
@@ -2,13 +2,105 @@
...
 title: Family Tree
-author: Danh Ta Chi Thanh
+author: Danh Ta Chi Thanh, Ryan Chou
 ---
 
 ## Solution
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_family_bronze_open18.html)
 
+By finding the closest common ancestor, and the distance from the ancestor and the two cows, we can uniquely identify their relationship.
```
Comment (maggie-j-liu) [inline solutions/usaco-833.mdx:12]:
```suggestion
By finding the closest common ancestor and the distances from the ancestor to the two cows, we can uniquely identify their relationship.
```

**Comment by HarryWangATX** [comment]
@ryanchou-dev http://bit.ly/3sbuO36 ;)

File: solutions/usaco-833.mdx [hunk 2]
```diff
@@ -2,13 +2,105 @@
...
+**Time Complexity:** $\mathcal{O}(N^3)$
+
+
+<LanguageSection>
+<PySection>
+	
+```py
+import sys
+
+sys.stdin = open('family.in', 'r')
```
Comment (maggie-j-liu) [inline solutions/usaco-833.mdx:23]: can you change the quotes to be consistent (either all single quotes or all double quotes)

File: solutions/usaco-833.mdx [hunk 3]
```diff
@@ -2,13 +2,105 @@
 id: usaco-833
 source: USACO Bronze 2018 Open
 title: Family Tree
-author: Danh Ta Chi Thanh
+author: Danh Ta Chi Thanh, Ryan Chou
 ---
 
 ## Solution
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_family_bronze_open18.html)
```
Comment (maggie-j-liu) [inline solutions/usaco-833.mdx:10]: since your solution is different from the alternative solution, put it in a separate languagesection

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: solutions/usaco-833.mdx (hunk 2 above)
Comment (ryanchou-dev) [reply solutions/usaco-833.mdx:23]: sure

**Comment by ryanchou-dev** [comment]
👍

**Review by maggie-j-liu — APPROVED** [review]

**Review by HarryWangATX — APPROVED** [review]
fill out the form 🙏

File: solutions/usaco-833.mdx [hunk 4]
```diff
@@ -2,13 +2,106 @@
...
+
+for i in range(int(n)):     
+	relations.append(input().split())
+
+# gets the mother of the cow (if it exists)
+def mother(cow):
+	for i in relations:
+		if i[1] == cow:
+			return i[0]
+	return ''
```
Comment (SansPapyrus683) [inline solutions/usaco-833.mdx:37]: maybe instead of an empty string return None instead?

File: solutions/usaco-833.mdx [hunk 5]
```diff
@@ -2,13 +2,106 @@
...
+if cow == '':
+	print('NOT RELATED')
+	sys.exit()
+
+# since we found a common ancestor, cow.
+# we can call direct_anc() to get the distance
+# from b to the common ancestor.
+db = direct_anc(cow, b)
+
+# if they're related by a common ancestor,
```
Comment (SansPapyrus683) [inline solutions/usaco-833.mdx:71]: when doing multiline comments,
```py
"""
can you style them like this
it looks better
"""
```

File: solutions/usaco-833.mdx [hunk 6]
```diff
@@ -2,13 +2,106 @@
...
+# if the distance from each cow to their common 
+# ancestor is one, they're siblings,
+elif da == 1 and db == 1:
+	print('SIBLINGS')
+	sys.exit()
+else:
+	if da > db:
+		da, db = db, da
+		b, a = a, b
+	print(a, 'is the ', end="")
```
Comment (SansPapyrus683) [inline solutions/usaco-833.mdx:87]: use single/double quotes consistently

File: solutions/usaco-833.mdx [hunk 7]
```diff
@@ -2,13 +2,106 @@
...
+	for _ in range(db - 2):
+		print("great-", end='')
+	if da == 0 and db > 1:
+		print('grand-', end='')
+	if da == 0:
+		print('mother ', end='')
+	else:
+		print('aunt ', end='')
+	print('of', b)
+
```
Comment (SansPapyrus683) [inline solutions/usaco-833.mdx:100]:
```suggestion
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
love your code, just some minor changes

File: solutions/usaco-833.mdx (hunk 6 above)
Comment (ryanchou-dev) [reply solutions/usaco-833.mdx:87]: oops, i missed that one. thanks for pointing it out!

**Comment by ryanchou-dev** [comment]
comments*

File: solutions/usaco-833.mdx (hunk 4 above)
Comment (ryanchou-dev) [reply solutions/usaco-833.mdx:37]: sure!

**Comment by ryanchou-dev** [comment]
👍 ?

**Review by SansPapyrus683 — APPROVED** [review]

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**